### DISCUSSION TITLE
How to disable flipper logs?

### DISCUSSION THREAD

**Discussion by IrakliJani** [jnunemaker/flipper#644] [Q&A] [answered]
Hey, Is there a way to disable flipper logs from the configuration?

**Comment by bkeepers** [comment, marked as answer]
@IrakliJani yep! Add this to your initializer:

```ruby
Rails.application.configure do
  config.flipper.log = false
end
```

**Comment by davidwessman** [comment]
```ruby
require "flipper/adapters/active_record"
require "flipper/adapters/active_support_cache_store"

Flipper.configure do |config|
  config.adapter do
    Flipper::Adapters::ActiveSupportCacheStore.new(
      Flipper::Adapters::ActiveRecord.new,
      Rails.cache,
      expires_in: 5.minutes
    )
  end
end
```



This is my configuration today, is there anyway for me to avoid Flipper-queries filling up my logs?
Such as `Flipper feature(account_merge) enabled? false (0.0ms)` and then all the actors passed.

**Reply by jnunemaker** [reply]
Yep. See https://github.com/flippercloud/flipper/issues/870

**Reply by davidwessman** [reply]
I tried to put it in the initialized instead of the rails configure 🙈
